lib/logflare/cache_buster.ex

```diff
@@ -20,9 +20,12 @@ defmodule Logflare.CacheBuster do
   end
 
   def handle_info(%Transaction{changes: changes}, state) do
-    for record <- changes do
-      handle_record(record)
+    for record <- changes,
+        record = handle_record(record),
```

Comment:
Can match on the tuple instead and remove the noop check.

Reply:
No, I cannot, as it would change behaviour (there will be match failure exception on `:noop`).